images/users/ark.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 260 45"><path fill="#5795AA" d="M79.33 19.127s-8.785-24.14-28.6-17.52c-8.88 2.967-13.253 14.347-23.505 15.718C16.04 18.822 2.405 6.8 0 2.96c0 0 1.654 2.818 4.31 6.692 13.01 20.253 21.408 33.102 35.406 34.723C57.272 46.408 66.584 29.4 68.942 22.59c1.954-5.645 5.724-5.428 8.145-4.576l2.243 1.113zM121.9 30.11h3.217V14.192H94.362V30.11h3.216l2.415 8.455h-9.06v3.965h37.614v-3.965h-9.06l2.414-8.455zm-22.096-3.964v-7.948h19.874v7.948H99.804zm14.257 12.42h-8.64l-2.41-8.456h13.478l-2.417 8.455zm14.28-32.1h-37.2v4.01h37.2v-4.01zM145.45 28l-2.666.35v-4.18h3.42v-3.832h-1.8l.66-7.713h.76V8.788h-3.197v-2.83h-3.982v2.83h-3.53v3.835h.697l.662 7.713h-1.864v3.832h3.814v4.296c0 .12-.008.28-.01.456l-3.767.494.46 4.07 3.003-.6c-.348 2.96-1.053 6.626-2.435 8.933l3.94 1.704c2.02-3.36 2.748-8.25 3.01-11.54l2.524-.5.29-3.472zm-4.097-7.662h-1.827l-.406-7.713h2.637l-.403 7.713zm30.464-7.714h1.34V8.788h-3.763v-2.83h-4.155v2.83h-3.96l-2.45-3.005-12.17 2.573v.624h-.01v13.1c0 8.185-.15 12.815-3.32 20.486l3.317.955c3.41-8.24 3.487-13.05 3.487-21.44V11.17l1.72-.26v27.845l-3.325.085.673 4.186 8.363-1.33-.61-3.073-2.05.052v-28.23l1.892-.287v11.92c0 3.898-.093 12.95 2.622 21.165l3.49-.712c-2.627-7.97-2.627-16.71-2.627-20.45V9.63l.85-.13v3.122h1.42l.66 7.713h-2.084v3.832h4.013v4.75h-3.51v3.832h3.51v10.29h4.33V32.75h3.422v-3.832h-3.427v-4.75h3.97v-3.832h-2.295l.664-7.714zm-3.712 7.714h-1.83l-.402-7.713h2.64l-.406 7.713zm26.37-9.323h16.168V37.63c0 1.236-1.06 1.236-1.55 1.236h-1.944l.693 4.104h3.393c1.02 0 4.13 0 4.13-3.772V7.072h-20.9v3.94zm-14.42 32.465h4.725V14.046h-4.727v29.432zm11.89-31.23L187.915 5l-4.743 2.25 4.91 6.44 3.86-1.445zm15.85 21.68s-1.42.862-3.604 1.537c-.73.225-1.35-.01-1.35-.985v-5.32h4.108V18.02h-4.656l2.924-4.51-3.27-1.937-4.18 6.447h-3.752l2.17-1.414-2.373-3.93-3.516 1.917 2.605 3.427h-4.8v11.14h3.3c.04 4.036-.57 6.302-4.3 8.406l2.474 3.68c5.097-3.028 6.2-6.878 6.368-12.087h2.92v5.43c-.028 4.51 3.088 6 5.47 4.9 1.147-.53 3.985-1.6 3.985-1.6l-.52-3.97zm-5.64-8.41h-9.263v-3.812h9.265v3.81zm52.4-2.013l4.024.647.95-5.342V14.94H248.43v-3.25h10.21V7.816h-10.21v-2.18h-4.208v2.18H233.93v3.874h10.297v3.248h-11.184v3.872H255.4l-.848 4.695zm4.775 16.052l-10.32-6.38h10.27V29.3H250c.44-1.956.505-3.538.505-4.263v-4.375h-4.266l-.008 4.375c0 .898-.115 2.47-.632 4.265h-3.914l.9-2.743-7.31-2.4-.792 2.725 5.004 2.413h-6.017v3.874h10.39c-1.696 2.67-4.585 5.185-9.484 6.31l.885 3.976c5.6-1.26 9.143-3.923 11.386-6.86l10.88 6.94.125.08 2.34-3.645-.67-.417zm-29.34-23.33h-8.56v4.05h4.35v17.74l-1.307 1.11 1.947 4.204 6.32-6.18v-5.1l-2.756 2.355V16.23zm1.9-4.86l-6.504-5.49-3.117 2.188 5.82 6.533 3.794-3.233zm11.68 14.15l1.21-3.657-7.313-2.395-.793 2.726 6.897 3.324z"/></svg>
```

Comment:
Thanks for your pr

Could you please resize your svg to 500x200?

Reply:
Done.